cdap-security/src/main/java/co/cask/cdap/security/runtime/AuthenticationServerMain.java

```diff
@@ -71,7 +73,12 @@ public void start() {
 
         Services.chainStart(zkClientService, authServer);
       } catch (Exception e) {
-        LOG.error("Got exception while starting authenticaion server", e);
+        Throwable rootCause = Throwables.getRootCause(e);
+        if (rootCause instanceof ServiceBindFailure) {
```

Comment:
Is there a reason we don't just check for a BindException here but translate the BindException into a ServiceBindFailure?  Is it because we don't have hostname/port info here?